bindings/python/crocoddyl/deprecated.py

```diff
@@ -30,3 +30,20 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return decorator
+
+
+class DeprecationHelper(object):
+    def __init__(self, new_target, old_name):
+        self.new_target = new_target
+        self.warning_str = f'{old_name} is deprecated: Use {new_target.__name__}'
```

Comment:
```suggestion
        self.warning_str = '%s is deprecated: Use %s' % (old_name, new_target.__name__)
```

Reply:
Sorry, but the f-strings are working only for Python >= 3.6

Reply:
thanks for the update